### PR TITLE
Fixes door pry spam

### PR DESCRIPTION
**Pull request by Cyberboss**
Re

### CODE REVIEW

**Comment by Armhulen** [comment]
NO

**Comment by PKPenguin321** [comment]
Sometimes, I pry on it so hard, I rip the skin!

**Comment by Qbopper** [comment]
this is a balance change because it makes me laugh and i dont want it merged

**Comment by Fox-McCloud** [comment]
`is it aliens or just the CE being a dick?`

Most definitely the latter.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -93,6 +93,7 @@ var/list/airlock_overlays = list()
 	hud_possible = list(DIAG_AIRLOCK_HUD)
 
 	var/air_tight = FALSE	//TRUE means density will be set as soon as the door begins to close
+	var/prying_so_hard = FALSE
```
Comment (PKPenguin321) [inline code/game/machinery/doors/airlock.dm]: hmm!

Comment (RemieRichards) [inline code/game/machinery/doors/airlock.dm]: I pried so hard, and got so far, but in the end this fucking airlock won't open.

**Pull request merged**